packages/airnode-protocol/contracts/rrp/authorizers/admin/interfaces/IAirnodeStablecoinPayment.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+interface IAirnodeStablecoinPayment {
+    event SetAirnodeFeeRegistry(
+        address airnodeFeeRegistry,
+        address registrySetter
+    );
+
+    event SetDefaultSupportedERC20(
+        uint256 chainId,
+        address stablecoin,
+        bool status,
+        address admin
+    );
+
+    event SetAirnodeSupportedERC20(
+        uint256 chainId,
+        address airnode,
+        address stablecoin,
+        bool status,
+        address admin
+    );
+
+    event SetSelfOptInStatus(address airnode, bool status);
+
+    event SetRequesterAuthorizerWithManager(
+        uint256 chainId,
+        address requesterAuthorizerWithManager,
+        address requesterAuthorizerWithManagerSetter
+    );
+
+    event SetAirnodePaymentAddress(address airnode, address paymentAddress);
+
+    event MadePayment(
+        uint256 chainId,
+        address indexed airnode,
+        bytes32 indexed endpointId,
+        address indexed requesterAddress,
+        address sponsor,
+        address paymentAddress,
+        uint256 paymentAmount,
+        uint64 expirationTimestamp
+    );
+
+    function setAirnodeFeeRegistry(address airnodeFeeRegistry) external;
+
+    function setRequesterAuthorizerWithManager(
+        uint256 chainId,
+        address requesterAuthorizerWithManager
+    ) external;
+
+    function setDefaultSupportedERC20(
+        uint256 chainId,
+        address stablecoin,
+        bool status
+    ) external;
+
+    function setAirnodeSupportedERC20(
+        uint256 chainId,
+        address airnode,
+        address stablecoin,
+        bool status
+    ) external;
+
+    function setSelfOptInStatus(address _airnode, bool _status) external;
```

Comment:
No, these were typos. I wanted to get the draft out to get feedback on the logic before I pushed the full PR and fixed the comments/typos. Especially because I couldn't get the stack too deep error resolved.